eksrollup/cli.py

```diff
@@ -261,6 +263,9 @@ def main(args=None):
     parser.add_argument('--plan', '-p', action='store_const', const=True,
                         help='perform a dry run to see which instances are out of date')
     args = parser.parse_args(args)
+
+    deployment_restart(node_name)
+
```

Comment:
What is this here? Just for debug I suppose..

Reply:
Huh this is odd. I must have accidentally commited this.